### DISCUSSION TITLE
How should i install extra cli tools, so that they are accessible in the workflow

### DISCUSSION THREAD

**Discussion by kaykhan** [actions/actions-runner-controller#3788] [FAQs]
If i want to have cli tools avaialble to use in the github runner. What is the best way to do this?

Is it possible to set a custom script .sh file to run?

**Comment by sjethvani** [comment]
>If i want to have cli tools avaialble to use in the github runner. What is the best way to do this?

Building custom docker image from base actions-runner image & use it when launching ephemeral runners using ARC
something like 
```
FROM ghcr.io/actions/actions-runner:latest

# Install required debian packages
RUN sudo apt-get update \
    && sudo apt-get install -y openssh-client zip git file jq curl \
    && sudo apt-get upgrade -y
```
>Is it possible to set a custom script .sh file to run?

Refer https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/running-scripts-before-or-after-a-job
